packages/components/inputs/SelectInput.tsx

```diff
@@ -224,9 +290,7 @@ const styles = StyleSheet.create({
     borderRadius: 6,
     padding: layout.padding_x1,
   },
-  // dropdownMenuRow: {
-  //   backgroundColor: neutral00,
-  //   borderRadius: 6,
-  //   padding: layout.padding_x1,
-  // },
+  inputIconContainer: {
+    marginLeft: layout.padding_x1_5,
+  },
```

Comment:
please don't use `stylesheet.Create` or add new fields in `stylesheet.Create`, this creates potential  undetectable deadcode and has no performance impact contrary to popular belief

if you want to define constant styles to prevent re-renders on memoized components, use something like
```ts
const inputIconContainerStyle: ViewStyle = {
  marginLeft: layout.padding_x1_5,
}
```